### PR TITLE
HtmlEditor - Fix validation's aria labels rendering

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js [hunk 1]
```diff
@@ -139,6 +139,21 @@ export default function() {
...
             }
         });
+
+        test('Content of editor should have aria-invalid attribute when validationStatus is invalid', function(assert) {
+            const $htmlEditor = $('#htmlEditor').dxHtmlEditor({
+                validationStatus: 'invalid',
+            });
+
+            const $editorContent = $htmlEditor.find(`.${HTML_EDITOR_CONTENT_CLASS}`);
+            const isQuillRendered = !!$editorContent.length;
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js:149]: In what case can content be not rendered?

File: packages/devextreme/js/ui/html_editor/ui.html_editor.js [hunk 2]
```diff
@@ -137,11 +137,18 @@ const HtmlEditor = Editor.inherit({
...
         this._renderSubmitElement();
+        this._markupInitialization = true;
         this.callBase();
-
+        this._markupInitialization = false;
         this._updateContainerMarkup();
     },
 
+    _renderValidationState() {
+        if(!this._markupInitialization) {
```
Comment (ksercs) [inline packages/devextreme/js/ui/html_editor/ui.html_editor.js]: cannot we get rid of this additional cache?
For example,  check that `this._ariaTarget()` exists

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js (hunk 1 above)
Comment (alexanderPolosatov) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js:149]: it's for server side tests. In this file, it is customary to write such checks

File: packages/devextreme/js/ui/html_editor/ui.html_editor.js (hunk 2 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/ui/html_editor/ui.html_editor.js]: reworked

**Review by marker-dao — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**